### DISCUSSION TITLE
RuntimeError: Inferred shape and existing shape differ in dimension 2: (16) vs (32)

### DISCUSSION THREAD

**Discussion by btwbtw01** [Xilinx/finn#575] [Q&A]
Hi,
I try to convert resnet20 model to hardware, but I got the error.
![image](https://user-images.githubusercontent.com/85877169/158607145-3f61f4fa-3dff-4ab2-9b8c-d3f9fda38428.png)
![image](https://user-images.githubusercontent.com/85877169/158607238-9ff9bf39-b3c5-4c79-8fc1-25cbbd5b8cfd.png)

I saw the ONNX graph, but I didn't see anything wrong. Maybe someone can solve it.
![end2end_resnet20_pre_post](https://user-images.githubusercontent.com/85877169/158607778-56c15722-d4e1-4af8-8eb1-b9056f08527d.png)
![end2end_resnet20_turn](https://user-images.githubusercontent.com/85877169/158607809-0da4ca76-c39c-4b0e-ba55-f3c28277f93d.png)

**Comment by o-ibrahim33** [comment]
@btwbtw01 do you have an answer yet? I am at the same problem and really stuck!
would appreciate the help
